packages/x-utils/src/__tests__/url.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { extractUrlParameters, getTaggingInfoFromUrl } from '../url';
+
+describe('url utils methods tests', () => {
```

Comment:
Please wrap same function test cases in their own `describe`, so it is easier to read which tests belong to which function:

```
describe('extractUrlParameters', () => {
  ....
};

...

describe('getTaggingInfoFromUrl', () => {
  ....
};
```